net/ddns-scripts/Makefile

```diff
@@ -50,6 +50,7 @@ endef
 define Package/ddns-scripts-services
   $(call Package/ddns-scripts/Default)
   TITLE:=Common ddns providers
+  PROVIDES:=ddns-scripts_service
```

Comment:
No need. I could not find any package in OpenWrt 19.07 with this naming. The same applies for OpenWrt 21.02.

Reply:
It seems like the package was added in 42696aa16234b466de54cde77fbef0220e5c3fff and renamed in a05030b9ca76c5c6bf73a7b8c720cb531fcf8853 which I guess happened only in master and never got to a release. I went through the renames in that PR and didn't check if it got to a release or not, I will check next time if you prefer to only handle renames in releases.